src/applications/mhv-medical-records/containers/App.jsx

```diff
@@ -125,7 +117,7 @@ const App = ({ children }) => {
     [user],
   );
 
-  if (featureTogglesLoading || user.profile.loading) {
+  if (user.profile.loading) {
```

Comment:
`state.user.profile.loading` is not the same as `state.featureToggles.loading`

Reply:
Just finished updating
###
<img width="1747" height="508" alt="Screenshot 2025-10-02 at 4 17 19 PM" src="https://github.com/user-attachments/assets/3c0d43c5-72b5-4a2c-97d1-bc96de001ec6" />